qrcode/android/src/main/kotlin/com/example/qrcode/QrcodePlugin.kt

```diff
@@ -24,7 +24,7 @@ class QrcodePlugin: FlutterPlugin, MethodCallHandler, ActivityAware {
     flutterActivityBinding = binding
     val qRCaptureViewFactory =
       flutterPluginBinding?.let { QRCaptureViewFactory(null, binding, binding.activity, it.binaryMessenger) }
-    flutterPluginBinding?.platformViewRegistry?.registerViewFactory("plugins/qr_capture_view", qRCaptureViewFactory);
+    flutterPluginBinding?.platformViewRegistry?.registerViewFactory("plugins/qr_capture_view", qRCaptureViewFactory!!);
```

Comment:
this might be safer

          qRCaptureViewFactory?.let { flutterPluginBinding?.platformViewRegistry?.registerViewFactory("plugins/qr_capture_view", it) }